packages/cmf/src/sagas/collection.js

```diff
@@ -0,0 +1,18 @@
+import { call, select } from 'redux-saga/effects';
+import { delay } from 'redux-saga';
+
+/**
+ * this saga ends when the collection is available
+ * @param {string} id of the collection to wait for
+ * @param {number} interval in ms
+ */
+export function* waitFor(id, interval = 10) {
+	// eslint-disable-next-line no-constant-condition
+	while (true) {
+		const collection = yield select(state => state.cmf.collections.get(id));
```

Comment:
Make a dediacted function instead of a anonymous one would be better no ?
By the way i think that you didn't do the TU here.